apis/opensearchservice/generator-config.yaml

```diff
@@ -0,0 +1,45 @@
+model_name: opensearch
+ignore:
+  resource_names:
+    - Package
+    # - Domain
+    - OutboundConnection
+  field_paths:
+    # SnapshotOptions is irrelevant for OpenSearch 5.3+
+    - CreateDomainInput.SnapshotOptions
+    - CreateDomainInput.Name
+    - DeleteDomainInput.Name
+resources:
+  Domain:
+    renames:
+      operations:
+        CreateDomain:
+          input_fields:
+            DomainName: Name
+            TagList: Tags
+        DeleteDomain:
+          input_fields:
+            DomainName: Name
+        DescribeDomain:
+          input_fields:
+            DomainName: Name
+    reconcile:
+      # Doing this because it takes a LONG time for the Domain's
+      # endpoint/endpoints field to be populated, even after the Domain's
+      # Processing field is set to False after creation...
+      requeue_on_success_seconds: 60
```

Comment:
is this (and the hooks) in use?

Reply:
Nope.  I removed both of them.  

They were copied over from the ack controller but are not used in the case of Crossplane code gen.